### PR TITLE
Clarify that the DID Method of a witness DID and diffentiating the DID Log Entry proofs

### PR DESCRIPTION
**Pull request by swcurran**
Addresses #114 and clarifies that a witness DID **SHOULD** be a did:tdw and why.  Also mentions the possiblity of mutual witnesses causing an infinite loop.

Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: spec/specification.md [hunk 1]
```diff
@@ -895,6 +903,12 @@ An overview of the [[ref: witness]] mechanism is as follows:
   - Over time, the list of [[ref: witnesses]] may evolve, with each change being
     approved by the declared list of [[ref: witnesses]] from **before** such a
     change.
+  - The DID of the witnesses **SHOULD** be `did:tdw` DIDs to prevent the need for
```
Comment (brianorwhatever) [inline spec/specification.md:906]: shouldn't this be a MUST?

File: spec/specification.md [hunk 2]
```diff
@@ -895,6 +903,12 @@ An overview of the [[ref: witness]] mechanism is as follows:
...
+    a `did:tdw` resolver having to resolve DIDs of other DID methods.
+  - `did:tdw` resolvers **SHOULD** account for the the possibility that the
```
Comment (brianorwhatever) [inline spec/specification.md:908]: how 🫠

Comment (swcurran) [reply spec/specification.md:912]: I would assume it is something like breaking recursion - record when you have seen a DID in the process, and break when you see the same DID again.  

That said, the spec. just has to point things out — it doesn’t have to do anything about it.  That’s why there are clever developers :-).

**Comment by swcurran** [comment]
@brianorwhatever  -- updated the DID, changing both SHOULDs to MUSTs and cleaned up the infinite loop wording.  I think the second MUST is also needed as a resolver that gets into an infinite loop isn't very useful.

**Comment by swcurran** [comment]
Given the planned redirection with witnesses per #135 -- closing this as overcome by events.

**Pull request closed without merging**